tests/Umbraco.Tests.Integration/Umbraco.Infrastructure/Services/MemberEditingServiceTests.cs

```diff
@@ -129,6 +131,7 @@ public async Task Can_Change_Member_Roles()
     {
         MemberService.AddRole("RoleTwo");
         MemberService.AddRole("RoleThree");
+        var grps = new[] { MemberGroupService.GetByName("RoleTwo"), MemberGroupService.GetByName("RoleThree") };
```

Comment:
grps? grapes? 🤣 I'm just kidding but being very pedantic I would really prefer `groups` 😄

Reply:
You are starting to sound like my old math teacher 🤣